src/runtime_src/core/pcie/windows/shim.cpp

```diff
@@ -753,16 +753,20 @@ struct shim
           return false;
       }
 
-      XOCL_PWRITE_BO_UNMGD_ARGS pwriteBO;
+      XOCL_PREAD_PWRITE_UNMGD_ARGS pwriteBO;
       DWORD  code;
       DWORD bytesWritten;
 
-      pwriteBO.Offset = offset;
+      pwriteBO.address_space = DDR_BANK0_ADDRESS;
+      pwriteBO.pad = DDR_PAD0;
```

Comment:
Hard code 0?

Reply:
Yes, it is. It is unused variable now but I was recommended to keep it.